index.js

```diff
@@ -8,20 +8,38 @@ const RedisStore = require('./store/RedisStore')
 
 const routeRateAdded = Symbol('fastify-rate-limit.routeRateAdded')
 
+const defaultHeaders = {
+  rateLimit: 'x-ratelimit-limit',
+  rateRemaining: 'x-ratelimit-remaining',
+  rateReset: 'x-ratelimit-reset',
+  retryAfter: 'retry-after'
+}
+
+const draftSpecHeaders = {
+  rateLimit: 'ratelimit-limit',
+  rateRemaining: 'ratelimit-remaining',
+  rateReset: 'ratelimit-reset',
+  retryAfter: 'retry-after'
+}
+
 function rateLimitPlugin (fastify, settings, next) {
   // create the object that will hold the "main" settings that can be shared during the build
   // 'global' will define, if the rate limit should be apply by default on all route. default : true
   const globalParams = {
     global: (typeof settings.global === 'boolean') ? settings.global : true
   }
 
+  const labels = (typeof settings.enableDraftSpec === 'boolean') ? draftSpecHeaders : defaultHeaders
```

Comment:
if the setting is explicitly set to false, won't this behave as if it was set to true?

Reply:
Yes, I think you are right . I will include it in the PR.

BTW similar behaviour is happening with the global parameter. https://github.com/fastify/fastify-rate-limit/blob/master/index.js#L15